.github/workflows/lint_pr.yml

```
@@ -0,0 +1,72 @@
+name: "gradido lint pull request CI"
+
+on:
+  pull_request:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # Configure which types are allowed (newline delimited).
+          # Default: https://github.com/commitizen/conventional-commit-types
+          #types: |
+          #  fix
+          #  feat
+          # Configure which scopes are allowed (newline delimited).
+          scopes: |
+            backend
+            wallet
```

```suggestion
            frontend
```

I would name it all over the place the same …